client/state/comments/schema.js

```diff
@@ -0,0 +1,83 @@
+export const commentsSchema = {
+	type: 'object',
+	patternProperties: {
+		'^\\d+-\\d+$': {
+			type: 'array',
+			items: {
+				type: 'object',
+				properties: {
+					ID: { type: 'integer' },
+					URL: { type: 'string' },
+					author: {
+						type: 'object',
+						properties: {
+							ID: { type: 'integer' },
+							URL: { type: 'string' },
+							avatar_URL: { type: 'string' },
+							email: { type: 'boolean' },
```

Comment:
hey let's not get too hung up on this either, instead focussing more energy on taking whatever needs @copons finds that we need to display the visual representation of a comment to lead us into a better state shape.